### PR TITLE
add tutorial for Kafka Connect support

### PR DESCRIPTION
**Pull request by arerlend**

### CODE REVIEW

**Comment by arerlend** [comment]
@basilhariri

File: tutorials/connect/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Kafka Connect support on Azure Event Hubs
```
Comment (basilhariri) [inline tutorials/connect/README.md]:
```suggestion
# Apache Kafka Connect support on Azure Event Hubs
```

File: tutorials/connect/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+## Prerequisites
+
+To complete this walkthough, make sure you have the following prerequisites:
+
+- Linux/MacOS
+
+- Kafka release (version 1.1.1, Scala version 2.11), available from [kafka.apache.org](https://kafka.apache.org/downloads#1.1.1)
+
+- Read through the [Event Hubs for Apache Kafka](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-for-kafka-ecosystem-overview) introduction article
```
Comment (basilhariri) [inline tutorials/connect/README.md]:
```suggestion
- Read through the [Event Hubs for Apache Kafka](https://docs.microsoft.com/azure/event-hubs/event-hubs-for-kafka-ecosystem-overview) introduction article
```

File: tutorials/connect/README.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+- Created [a Kafka-enabled Event Hubs namespace](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create) on the Azure Portal
```
Comment (basilhariri) [inline tutorials/connect/README.md]:
```suggestion
- Created a [Kafka-enabled Event Hubs namespace](https://docs.microsoft.com/azure/event-hubs/event-hubs-create) on the Azure Portal
```

File: tutorials/connect/README.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+[Kafka Connect](https://kafka.apache.org/documentation/#connect) is a platform that provides easy integration of Apache Kafka and other services into secure and scalable pipelines.  Workers can be deployed in standalone mode (single processes) or distributed mode (nodes in a cluster of Connect workers).  Source and sink connectors are used to define movement of data in and out of the Kafka cluster respectively.
+
+More information on Kafka Connect concepts is available [here](https://docs.confluent.io/current/connect/concepts.html).
+
+## Configuring Kafka Connect for Event Hubs
+
+Minimal reconfiguration is necessary when redirecting Kafka Connect throughput from Kafka to Event Hubs.  The following `connect-distributed.properties` sample illustrates how to configure Connect to authenticate and communicate with the Kafka endpoint on Event Hubs:
+
+```properties
+bootstrap.servers={YOUR.EVENTHUBS.HOST}:9093 # e.g. namespace.servicebus.windows.net:9093
```
Comment (basilhariri) [inline tutorials/connect/README.md]:
```suggestion
bootstrap.servers={YOUR.EVENTHUBS.FQDN}:9093 # e.g. namespace.servicebus.windows.net:9093
```

File: tutorials/connect/README.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+2. Navigate to the location of the Kafka release on your machine.
+
+3. Run `./bin/connect-distributed.sh /PATH/TO/connect-distributed.properties`.  The Connect worker REST API is ready for interaction when you see `'INFO Finished starting connectors and tasks'`. 
+
+### Creating Connectors
+
+This section will walk you through spinning up FileStreamSource and FileStreamSink connectors. 
+
+1. Create a directory for our input and output data files to live
```
Comment (basilhariri) [inline tutorials/connect/README.md]:
```suggestion
1. Create a directory for our input and output data files to live in
```

File: tutorials/connect/README.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+### Cleanup
+
+Kafka Connect will create Event Hub topics to store configurations, offsets, and status that persist even after the Connect cluster has been taken down.  Unless this persistence is desired, it is recommended that these topics are deleted.  You may also want to delete the `connect-quickstart` Event Hub that were created during the course of this walkthrough.
+
+You can use any of the following tools to manage your Event Hubs namespace:
+
+- [Azure portal](portal.azure.com) (Note: consumer groups auto-created by Kafka clients will not appear on the Event Hubs consumer group lists.)
+
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/eventhubs?view=azure-cli-latest)
```
Comment (basilhariri) [inline tutorials/connect/README.md]:
```suggestion
- [Azure CLI](https://docs.microsoft.com/cli/azure/eventhubs?view=azure-cli-latest)
```

File: tutorials/connect/README.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+- [C# Namespace Manager library](https://docs.microsoft.com/en-us/dotnet/api/microsoft.servicebus.namespacemanager?view=azure-dotnet)
```
Comment (basilhariri) [inline tutorials/connect/README.md]:
```suggestion
- [C# Namespace Manager library](https://docs.microsoft.com/dotnet/api/microsoft.servicebus.namespacemanager?view=azure-dotnet)
```

**Pull request merged**